setup.sh

```diff
@@ -20,35 +20,36 @@ osx() {
 
 install_apt() {
     sudo apt-get update || true
-    sudo apt-get -y install git gdb python3-dev python3-pip python3-setuptools libglib2.0-dev libc6-dbg
+    sudo apt-get install -y git gdb python3-dev python3-pip python3-setuptools libglib2.0-dev libc6-dbg nasm
+    test -f /usr/bin/go || sudo apt-get install -y golang
```

Comment:
It looks weird but GitHub set up the Ubuntu's container and it has installed `go` v1.14 when we execute `apt-get install` it re-installs `go` v1.10 but it didn't override the variable environments, the execution of the tests fails. Only install if `go` binary, not exists.

Reply:
I think we need to have a separate "install test dependencies" thing. I mean, we don't want to require users to install Go or nasm in order to run (and not test) Pwndbg

Reply:
This being said, I don't know how to do it well cross platform (the package names may differ). I think we can stay ubuntu-only for tests?

Reply:
> This being said, I don't know how to do it well cross platform (the package names may differ). I think we can stay ubuntu-only for tests?

I'm totally agreed to keep one distribution for testing and it could be Ubuntu, since your CI workflows match with this.

Reply:
> I think we need to have a separate "install test dependencies" thing. I mean, we don't want to require users to install Go or nasm in order to run (and not test) Pwndbg

Done in the commit 60c0283c321a788e8a9fb3e0eefafc91b414e88f

Reply:
Added other commit related to this discussion: commit ae5905e49d5e9416056684efda1e7b54b8747dea